assets/src/blocks/ENForm/ENFormEditor.js

```diff
@@ -7,10 +6,13 @@ import { useSelect } from '@wordpress/data';
 
 
 export const ENFormEditor = ({ attributes, setAttributes, isSelected }) => {
+  return (
```

Comment:
You can just pass the `attributes` object like { ...attrValues } instead of passing `{ attributes: { ...attrValues } }`

And then 
```
const renderEdit = (attributes, setAttributes) => {
  const { en_form_style, className, background, background_image_src } = attributes;
```